plutus-core/plutus-core/src/PlutusCore/Builtin/Meaning.hs

```diff
@@ -1,6 +1,7 @@
 -- GHC doesn't like the definition of 'makeBuiltinMeaning'.
 {-# OPTIONS_GHC -fno-warn-redundant-constraints #-}
 
+{-# LANGUAGE AllowAmbiguousTypes       #-}
```

Comment:
do we need this?

Reply:
I see, it's because we dropped the `Proxy` args. Does that matter? Perhaps this change should be done separately so we can see if it matters.

Reply:
> Does that matter?

Probably not. I did it to see if it could change Core output, but couldn't tell if there was a difference in the end. I left it in the code, because `AllowAmbiguousTypes` looks better to my taste than those stupid proxies.